.gitignore

```diff
@@ -2,3 +2,4 @@
 .gitignore
 yarn-error.log
 node_modules/
+.idea
```

Comment:
Is this intentional?

Reply:
Yes it is because I am working with intellij and it is saving their local settings. But it will hurt no one.

Reply:
👍 
did you "resolve" this thread? if so, please leave that to the person that started the thread. if not, I need to understand GH better :-)